pkg/controller/resourceconfig/static_config.go

```diff
@@ -0,0 +1,1684 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resourceconfig
+
+import "github.com/GoogleCloudPlatform/k8s-config-connector/pkg/k8s"
+
+var ControllerConfigStatic = &ResourcesControllerMap{
+	{Group: "storage.cnrm.cloud.google.com", Kind: "StorageBucketAccessControl"}: {
+		DefaultController:    k8s.ReconcilerTypeTerraform,
+		SupportedControllers: []k8s.ReconcilerType{k8s.ReconcilerTypeTerraform},
+	},
+	{Group: "monitoring.cnrm.cloud.google.com", Kind: "MonitoringUptimeCheckConfig"}: {
```

Comment:
Would also be good to sort this file by group then kind. Stable order will both make it easier to find things, minimize collisions in PRs and minimize difficult PR reviews.